File: assessment/src/main/java/com/mediscreen/assessment/dto/Assessment.java
package com.mediscreen.assessment.dto;

import com.mediscreen.assessment.model.RiskLevel;
import com.mediscreen.assessment.model.TriggerTerms;
import lombok.Builder;
import lombok.Value;

import java.util.List;

@Value
@Builder
public class Assessment {

    Long patientId;

    String patientLastName;

    String patientFirstName;

    int age;

    RiskLevel riskLevel;

    List<TriggerTerms> terms;
}

File: medapp/src/app/interface/assessment.ts
import { RiskLevel } from "../enum/risk-level.enum";
import { TriggerTerms } from "../enum/trigger-terms.enum";

export interface Assessment {
    id: number;
    patientLastName: string;
    patientFirstName: string;
    age: number;
    riskLevel: RiskLevel;
    terms: TriggerTerms[];
}